src/Explorer/Tree/Database.ts

```diff
@@ -388,4 +396,39 @@ export default class Database implements ViewModels.Database {
   private _getOfferForDatabase(offers: DataModels.Offer[], database: DataModels.Database): DataModels.Offer {
     return _.find(offers, (offer: DataModels.Offer) => offer.resource === database._self);
   }
+
+  private addSchema(collection: DataModels.Collection): void {
+    if (collection.analyticalStorageTtl == undefined || !this.container.isSchemaEnabled()) {

```

Comment:
Is there a particular reason you use `== undefined` instead of `=== undefined`?

Reply:
nope, c# muscle memory .